network/ServerNetworking.h

```diff
@@ -326,6 +326,7 @@ class PlayerConnection :
     bool                            m_authenticated;
     Networking::AuthRoles           m_roles;
     boost::uuids::uuid              m_cookie;
+    bool                            m_valid;
```

Comment:
no need for threaded access guards?

Reply:
Server uses boost asio in a single threaded event loop.